### DISCUSSION TITLE
Run build creates a single image with all the layers from each folder

### DISCUSSION THREAD

**Discussion by cyclez** [nftchef/art-engine#285] [Help + Q&A]
Hey there. `npm run build` creates only a single image with all the layers inside:

genator started undefined
Editions left to create:  [ 0, 1 ]
DNA: [
  '0.4:Wall.png',
  '0.3:Neon_Street.png',
  '0.2:Dragon_Neon.png',
  '0.1:Construction_Site.png',
  '0.0:Barbed_Wire.png',
  '1.3:AccSpalla_4.png',
  '1.2:AccSpalla_3.png',
  '1.1:AccSpalla_2.png',
  '1.0:AccSpalla_1.png',
  '2.4:Sand.png',
  '2.3:Pink.png',
  '2.2:Brown.png',
  '2.1:Black.png',
  '2.0:Albino.png',
  '3.4:Eyes_5.png',
  '3.3:Eyes_4.png',
  '3.2:Eyes_3.png',
  '3.1:Eyes_2.png',
  '3.0:Eyes_1.png',
  '4.4:Lips_5.png',
  '4.3:Lips_4.png',
  '4.2:Lips_3.png',
  '4.1:Lips_2.png',
  '4.0:Lips_1.png',
  '5.4:mu_5.png',
  '5.3:mu_4.png',
  '5.2:mu_3.png',
  '5.1:mu_2.png',
  '5.0:mu_1.png',
  '6.3:Tattoo_4.png',
  '6.2:Tattoo_3.png',
  '6.1:Tattoo_2.png',
  '6.0:Tattoo_1.png',
  '7.4:Piercing_5.png',
  '7.3:Piercing_4.png',
  '7.2:Piercing_3.png',
  '7.1:Piercing_2.png',
  '7.0:Piercing_1.png',
  '8.2:AccPolsoDX_3.png',
  '8.1:AccPolsoDX_2.png',
  '8.0:AccPolsoDX_1.png',
  '9.1:AccPolsoSX_2.png',
  '9.0:AccPolsoSX_1.png',
  '10.4:Veste_5.png',
  '10.3:Veste_4.png',
  '10.2:Veste_3.png',
  '10.1:Veste_2.png',
  '10.0:Veste_1.png',
  '11.4:Hair_5.png',
  '11.3:Hair_4.png',
  '11.2:Hair_3.png',
  '11.1:Hair_2.png',
  '11.0:Hair_1.png',
  '12.4:AccManoDX_5.png',
  '12.3:AccManoDX_4.png',
  '12.2:AccManoDX_3.png',
  '12.1:AccManoDX_2.png',
  '12.0:AccManoDX_1.png',
  '13.5:AccManoSX_6.png',
  '13.4:AccManoSX_5.png',
  '13.3:AccManoSX_4.png',
  '13.2:AccManoSX_3.png',
  '13.1:AccManoSX_2.png',
  '13.0:AccManoSX_1.png'
]
(node:40498) [DEP0147] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will be removed. Use fs.rm(path, { recursive: true }) instead
(Use `node --trace-deprecation ...` to show where the warning was created)

then i get `DNA exists !` error

All i did was replace the layers order from the original config.js with my actual folders.

Any guess of what is going wrong? thank you.

**Comment by VanillaBiscuit** [comment]
okay bud wild thought, see I had a similar issue, but your post clarified a lot to me, see if this works and give me a reply here. 

Try adding # weights to your files EX. img#10.png and see if they generate correctly, I personally ran into a similar problem, the cause may be different but try it out, who knows. :) 

I suspect the files need # weights without exception

**Comment by cyclez** [comment]
> okay bud wild thought, see I had a similar issue, but your post clarified a lot to me, see if this works and give me a reply here.
> 
> Try adding # weights to your files EX. img#10.png and see if they generate correctly, I personally ran into a similar problem, the cause may be different but try it out, who knows. :)
> 
> I suspect the files need # weights without exception

inserting weights helped with some folder but not with others, trying to mess around with it

**Reply by dzmbs** [reply]
you found solution for this? having same problem

**Comment by VanillaBiscuit** [comment]
glad it helped with some of them 🤟, I can't think of anything else that would cause it
